.github/workflows/main.yml

```diff
@@ -527,6 +580,18 @@ jobs:
           python -m pip install --upgrade -r ./build/conda-functional-requirements.txt
           python -m ipykernel install --user
 
+      - name: Install Python Libs for conda
+        shell: bash -l {0}
+        if: matrix.test-suite == 'conda'
+        run: |
+          conda activate functional_test_env
```

Comment:
Doesn't seem like a bit deal either way, but just out of interest. Looks like that Conda github action is already running an activate-environment step, do we need this activate here?

Reply:
Yes. In github actions each step is a different bash shell.